oeps/processes/oep-0021-proc-deprecation.rst

```diff
@@ -274,7 +294,7 @@ Do the following to document your proposal:
    #. **Removal**: A description with links to what is being removed.
    #. **Replacement**: A description with links to what it is being replaced by.
    #. If you plan to mark the code for deprecation, explain how in the
-      **Deprecation** section. See Deprecated_ for considerations.
+      **Deprecation** section. See Deprecated for considerations.
```

Comment:
Rather than just dropping the link from "See Deprecated_ for considerations", I think this whole sentence needs to be updated or removed.